### PR TITLE
Set ref torque group

### PR DESCRIPTION
**Pull request by barbalberto**
Implements setRefTorque method for group of joints.
Increased yarp version  number dependency.

### CODE REVIEW

File: src/tools/simpleClient/main.cpp [hunk 1]
```diff
@@ -932,6 +933,45 @@ void handleTorqueMsg(ITorqueControl *torque, const yarp::os::Bottle& cmd,
...
+                            }
+
+                            if (n_joint > controlledJoints)
+                            {
+                                yError() << "command malformed, number of joints inserted is bigger than number of available joints (" << controlledJoints << ")";
+                                *rec = false;
+                                break;
+                            }
+
+                                int    *joints = new int[n_joint];       // LATER: optimize to avoid allocation.
```
Comment (traversaro) [inline src/tools/simpleClient/main.cpp]: Why later? I guess having a buffer big enough will be sufficient for all `n_joint`, given that the size of the buffer is specified as a specific `n_joint` argument of the setRefTorques call:
~~~
*ok = torque->setRefTorques(n_joint, joints, refs); 
~~~
So even if joints is a vector of size 1000, there are no problems.

Comment (traversaro) [inline src/tools/simpleClient/main.cpp]: Sorry, I did not notice that this is simply the simpleClient... then is ok!

**Comment by barbalberto** [comment]
Si può far ripartire il test?

**Comment by francesco-romano** [comment]
I restarted the travis build

**Pull request merged**